modules/ROOT/pages/_partials/task-add-api-instance.adoc

```diff
@@ -33,4 +33,8 @@ instance.
 
 When you are promoting or importing an API instance, you do not have options to alter the configuration. However, when you add a new API instance, you need to the downstream and the upstream configuration settings.
 
+The following diagram demonstrates the upstream and downstream configurations settings relationship to your upstream service and consumer application.
```

Comment:
Suggestion: ...configures the relationship between the upstream and downstream configuration settings to your upstream service and consumer application.

Reply:
The way you have it written, you would need to change "configurations settings relationship" to "configuration settings' relationship." This wording is awkward.